.github/workflows/test.yml

```diff
@@ -84,6 +84,11 @@ jobs:
       - name: Install Dependencies ⬆️
         run: yarn install --immutable
 
+      # Build the design tokens. Our Tailwind config imports these, and linting will fail if not
+      # present.
+      - name: Build design tokens (needed for linting to succeed)
+        run: yarn build:tokens
```

Comment:
Investigating what the heck is going on here.

Linting failed on CI because:
- tailwind.config.js imports tokens
- If the tokens aren't present, linting fails

Adding this here ensures the tokens are present.

No idea why this wasn't an issue until now, though... 😬🤔